src/Utils/Image.cs

```diff
@@ -280,7 +281,11 @@ public Image ConvertTo(string format, string metadata = null, int dpi = 0)
         switch (format)
         {
             case "PNG":
-                PngEncoder encoder = new() { TextCompressionThreshold = int.MaxValue };
+                PngEncoder encoder = new()
+                {
+                    TextCompressionThreshold = int.MaxValue,
+                    BitDepth = img.PixelType.BitsPerPixel == 8 ? PngBitDepth.Bit8 : PngBitDepth.Bit16
```

Comment:
Should swap this check - `== 16 ? Bit16 : Bit8` - ie for any unexpected value, default to 8

Reply:
Good catch